test/libsolidity/smtCheckerTests/operators/const_exp_1.sol

```diff
@@ -0,0 +1,15 @@
+pragma experimental SMTChecker;
+
+contract C {
+	uint constant x = 2;
+	uint constant y = x ** 10;
+
+	function f() public view {
+		assert(y == 2 ** 10);
+		assert(y == 1024);
+		assert(y == 14);
```

Comment:
```suggestion
		assert(y == 14); // Should fail
```

I know the cex explains it well, but people may just blindly update the cex in error, and then this information is "lost".

Reply:
Yep!